setupext.py

```diff
@@ -432,10 +433,10 @@ def get_extensions(self):
         add_libagg_flags_and_sources(ext)
         yield ext
         # qhull
-        ext = Extension(
+        ext = Pybind11Extension(
             "matplotlib._qhull", ["src/_qhull_wrapper.cpp"],
+            cxx_std=11,
```

Comment:
We can specify the C++ standard to follow, so here we are explicitly restricting to C++11. This would probably be a configuration param at the top of the file to make it clear what C++ standard the project is currently working to.